TLM/TLM/UI/TrafficManagerTool.cs

```diff
@@ -442,13 +511,41 @@ public class TrafficManagerTool
                 if (_activeSubTool != null) {
                     _activeSubTool.OnToolGUI(e);
                 } else {
-                    base.OnToolGUI(e);
+                    DefaultOnToolGUI(e);
                 }
             } catch (Exception ex) {
                 Log.Error("GUI Error: " + ex);
             }
         }
 
+        void DefaultOnToolGUI(Event e) {
+            if (e.type == EventType.MouseDown && e.button == 0) {
+                bool isRoad = HoveredSegmentId != 0 && HoveredSegmentId.ToSegment().Info.m_netAI is RoadBaseAI;
+                if (!isRoad)
+                    return;
+
+                if (ReadjustPathMode) {
+                    bool isRoundabout = RoundaboutMassEdit.Instance.TraverseLoop(HoveredSegmentId, out var segmentList);
+                    if (!isRoundabout) {
+                        segmentList = SegmentTraverser.Traverse(
+                            HoveredSegmentId,
+                            TraverseDirection.AnyDirection,
+                            TraverseSide.Straight,
+                            SegmentStopCriterion.None,
+                            (_) => true);
+                    }
+                    RoadSelectionUtil.SetRoad(HoveredSegmentId, segmentList);
```

Comment:
Nullpointer possible, `segmentList` can be null, because of changes to `Segment.Traverser()` in this PR